src/hocon_pp.erl

```diff
@@ -154,7 +155,7 @@ maybe_quote(K) when is_atom(K) -> atom_to_list(K);
 maybe_quote(K) ->
     case re:run(K, "[^A-Za-z_]") of
         nomatch -> K;
-        _ -> io_lib:format("~100000p", [unicode:characters_to_list(K, utf8)])
+        _ -> io_lib:format("~0p", [unicode:characters_to_list(K, utf8)])
```

Comment:
As from Erlang/OTP 21.0, a field width of value 0 can be used for specifying that a line is infinitely long, which means that no line breaks are inserted. For example:
```
1> io:fwrite("~0p~n", [lists:seq(1, 30)]).
[1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30]
ok
```